test/util/zip.js

```diff
@@ -91,21 +91,26 @@ const binaryParser = (res, callback) => {
   });
 };
 
-const httpZipResponseToFiles = (zipHttpResponse) => new Promise((resolve, reject) => {
-  zipHttpResponse.buffer().parse(binaryParser).end((err, res) => {
-    if (err) return reject(err);
-
-    // eslint-disable-next-line no-shadow
-    yauzl.fromBuffer(res.body, (err, zipfile) => {
+const httpZipResponseToFiles = (zipHttpResponse) => {
+  zipHttpResponse
+    .expect(200)
```

Comment:
I didn't understand at first how this would cause a non-200 to lead to a test failure, given that this is separate from the promise below. But then I realized that `zipHttpResponse` is mutable, and calling `.expect(200)` here causes `.buffer()` to fail below if the response is not a 200.

Reply:
Would this be more idiomatic?

```js
const httpZipResponseToFiles = (zipHttpResponse) => new Promise((resolve, reject) => {
  zipHttpResponse
    .expect(200)
    .expect('Content-Type', 'application/zip')
    .buffer().parse(binaryParser).end((err, res) => {
```

Reply:
I like how that looks! I don't have an issue with the other way either if you prefer that.